ros2bag/package.xml

```diff
@@ -2,7 +2,7 @@
 <?xml-model href="http://download.ros.org/schema/package_format2.xsd" schematypens="http://www.w3.org/2001/XMLSchema"?>
 <package format="2">
   <name>ros2bag</name>
-  <version>0.2.3</version>
+  <version>0.2.4</version>
```

Comment:
Shouldn't version bump be reserved for the release process? Same for the other package.xmls

Reply:
yeah git suggested the signoff and we did it without caution. this will probably get reverted tomorrow (it's 10pm here ^-^)